src/test/java/org/example/homework13/blackjack/DeckTest.java

```diff
@@ -0,0 +1,38 @@
+package org.example.homework13.blackjack;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+class DeckTest {
+
+    @Test
+    void drawCard() {
+        Deck deck = new Deck();
+        Card card = deck.drawCard();
+        assertEquals(Rank.TWO, card.getRank());
+        assertEquals(Suit.CLUBS, card.getSuit());
+    }
+
+    @Test
+    void shuffle() {
+        Deck deck = new Deck();
+        List<Card> referenceList = new ArrayList<>(deck.getCards());
+        deck.shuffle();
+        assertNotEquals(referenceList, deck.getCards(), "Deck was not shuffled");
```

Comment:
only assert equals is not enough. ideally do something like this
```suggestion
        var deck = new Deck();
        List<Card> originalCards = new ArrayList<>(deck.getCards());
        deck.shuffle();
        List<Card> shuffledCards = deck.getCards();
        assertEquals(originalCards.size(), shuffledCards.size(),
                "The size of the deck should remain unchanged");
        assertNotEquals(originalCards, shuffledCards,
                "The deck should be shuffled and not in original order");
        assertTrue(shuffledOrder.containsAll(originalCards) && originalCards.containsAll(shuffledCards),
                "The shuffled deck should contain all original cards");```